.github/workflows/release.yaml

```diff
@@ -0,0 +1,29 @@
+name: Release charm to other tracks and channels
+
+on:
+  workflow_dispatch:
+    inputs:
+      destination-channel:
+        description: 'Destination Channel'
+        required: true
+      origin-channel:
+        description: 'Origin Channel'
+        required: true
```

Comment:
According to the release template, origin channel is required for single charm repos, but not for multi ones. @agathanatasha do you know if there a reason behind this difference?